SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -10,6 +10,8 @@
 #include "Settings/LevelEditorPlaySettings.h"
 #endif
 
+DEFINE_LOG_CATEGORY(LogSpatialGdkSettings);
```

Comment:
nit: can this be LogSpatialGDKSettings ?

Reply:
done